betfund_event_broker/flows/base_flow.py

```diff
@@ -37,7 +37,7 @@ class EventBrokerFlow(ABC):
 
     def __init__(
         self,
-        distributed: bool = False,
+        distributed: bool = True,
```

Comment:
Default to `True` here?

Reply:
Good catch - will fix (did that for some testing on CLI)